File: Java/Rango.java
import java.io.*;

public class Rango {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    public static void main(String[] args) throws Exception {

        int num1 = Integer.parseInt(br.readLine());
        int num2 = Integer.parseInt(br.readLine());

        System.out.println((num1 > num2) ? "El primer valor tiene que ser menor." : "");

        for (int i = num1; i <= num2; i++) {
            System.out.print("[" + i + "]");
        }
    }
}

File: Python/Rango.py
def main():
    num1 = int(input())
    num2 = int(input())

    if num1 > num2: 
        print("El primer numero tiene que ser menor.")
    else:
        for i in range(num1, num2 + 1):
            print(f"[{i}]", end=" ")

main()

# https://omegaup.com/arena/problem/C40/